config.py

```diff
@@ -267,7 +275,7 @@ def load(cls, _db=None):
         """Load configuration information from the filesystem, and
         (optionally) from the database.
         """
-        cls.instance = cls.load_from_file(_db)
+        cls.instance = cls.load_from_file()
```

Comment:
An unrelated change, fallout from my earlier branch that changed the way `Configuration` loading worked.